torch/_inductor/config.py

```diff
@@ -316,6 +316,12 @@ def bundle_triton_into_fx_graph_cache_default() -> Optional[bool]:
 # default value is InfiniBand
 inter_node_bw = 25
 
+# use Inductor's experimental benchmarker (runtime/benchmarking.py)
+# to benchmark kernels during autotuning, otherwise fall back to
+# Triton's `do_bench`. the experimental benchmarker may produce
+# results that are not consistent with `do_bench`'s results
+use_experimental_benchmarker = True
```

Comment:
nit - use this [Config](https://github.com/pytorch/pytorch/blob/3908be676c7406da52824c83b804ae45e1daff7a/torch/utils/_config_module.py#L47) class that will JK if we need it.